coderd/organizations.go

```diff
@@ -95,12 +94,11 @@ func (api *API) postOrganizations(rw http.ResponseWriter, r *http.Request) {
 			UserID:         apiKey.UserID,
 			CreatedAt:      dbtime.Now(),
 			UpdatedAt:      dbtime.Now(),
-			Roles: []string{
+			Roles:          []string{
 				// TODO: When organizations are allowed to be created, we should
 				// come back to determining the default role of the person who
 				// creates the org. Until that happens, all users in an organization
 				// should be just regular members.
```

Comment:
What about this TODO item? Out of scope?

Reply:
Yup out of scope. No one has requested this yet, but I imagine with custom roles it'll be asked